Resources/views/snippets/pkgShop/shopBasket/shopBasketMiniBasket.html.twig

```diff
@@ -24,7 +24,7 @@
     <div class="shopBasketMiniBasketBox">
         <div class="shopBasketMiniBasketBoxBoxInner">
             <div class="shopBasketMiniBasketBoxBoxSpacerRight">
-                <a class="shopBasketMiniBasketBoxBoxSpacerLeft teaser" href="{{sBasketURL|raw}}" title="{{"chameleon_system_chameleon_shop_theme.minibasket.action_show_basket_title"|trans}}">
+                <a class="shopBasketMiniBasketBoxBoxSpacerLeft teaser" href="[{basketUrl}]" title="{{"chameleon_system_chameleon_shop_theme.minibasket.action_show_basket_title"|trans}}">
```

Comment:
I don't know why it was sent through the raw filter before, but in general we should escape values we put into attributes using `| e('html_attr')`

Reply:
Scratch that! We of course do not have the filters using our own template substitution here.